LocalWiki.php

```diff
@@ -602,7 +602,6 @@ function onSkinAddFooterLinks( Skin $skin, string $key, array &$footerItems ) {
 				);
 			}
 		}
-
```

Comment:
Lol sorry I wasn't clear, readd the line but remove the extra tabbing you had so this has no change being displayed at all.